docs/architecture/grpc-for-wcf-developers/protobuf-maps.md

```diff
@@ -44,6 +44,8 @@ public Order CreateOrder(Dictionary<string, string> attributes)
 
 For more information about Protobuf, see the official [Protobuf documentation](https://developers.google.com/protocol-buffers/docs/overview).
 
+[map-field]: https://developers.google.cn/protocol-buffers/docs/reference/csharp/class/google/protobuf/collections/map-field-t-key-t-value-
```

Comment:
https://review.docs.microsoft.com/en-us/dotnet/architecture/grpc-for-wcf-developers/protobuf-maps?branch=pr-en-us-19908